fab/environments.yml

```diff
@@ -143,7 +143,7 @@ production:
       email_queue:
         concurrency: 2
       repeat_record_queue:
-        concurrency: 6
+        concurrency: 20
```

Comment:
agreed i was seeing some weird things happen when i'd bump the concurrency too high